layouts/shortcodes/cloud-intro.html

```diff
@@ -2,7 +2,8 @@
     <p>
         Pulumi's infrastructure-as-code SDK helps you create, deploy, and manage {{ .Get 0 }} containers,
         serverless functions, and infrastructure using real programming languages. The Pulumi {{ .Get 0 }} provider packages
-        and <a href="{{ relref . "/docs/reference/cli" }}">CLI</a> help you accomplish all these within minutes.
+        and <a href="{{ relref . "/docs/reference/cli" }}">CLI</a>
+        help you accomplish all these within minutes.
```

Comment:
Outside this PR, but since we're in there:

```suggestion
        help you accomplish all of these within minutes.
```